tgm/nn/model/dygformer.py

```diff
@@ -0,0 +1,585 @@
+"""Adapted from https://github.com/yule-BUAA/DyGLib_TGB."""
+
+from typing import Callable, Tuple
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..time_encoding import Time2Vec
+
+PADDED_NODE_ID = -1
```

Comment:
We have it now in our constants module :)

```suggestion
from tgm.constants import PADDED_NODE_ID
```